extension/src/util/string.ts

```diff
@@ -1 +1,8 @@
-export const shortenForLabel = (str: string): string => str.slice(0, 7)
+export const shortenForLabel = <T extends string | null>(
+  strOrNull: T
+): T | string => {
+  if (typeof strOrNull === 'string') {
+    return strOrNull?.slice(0, 7)
```

Comment:
Why not simply check for `strOrNull` to be truthy instead of checking for the type to be a string? A conditional wouldn't be necessary here.

Reply:
![image](https://user-images.githubusercontent.com/37993418/185251845-5d02e4c4-5dc6-4f97-adec-781d393d3819.png)

Will use a ternary.